### DISCUSSION TITLE
关于页面存在多个编辑器，上一个编辑器下拉菜单会被下一个编辑器的头部菜单栏所遮挡的问题

### DISCUSSION THREAD

**Discussion by 123lujiaqiang** [wangeditor-team/wangEditor#3953] [Q&A]
我在Dialog弹出框中存在多个编辑器（公共组件），当使用第一个编辑器，打开一个下拉框时，下拉菜单会被下一个编辑器的头部菜单栏所遮挡（下拉菜单是自定义的公式编辑器），我认为这是因为编辑器头部菜单栏层级太高的问题，我也在找到了好多关于层级覆盖的问题及解决方案，但是对我却没有用。
最后，我在editor的配置中，设置了editor.config.zIndex=-1;(官方文档说明中：zindex>=0)，这是因为当我设置zlndex=0时，包裹编辑器头部菜单栏的div会默认存在一个z-index=1；我查看了编辑器的源码，我发现会自动默认设置一个z-index的值是10000，但是，包裹编辑器头部菜单栏的div总会在这个基础上+1，也就是z-index会变成10001。
我将它的值设置为editor.config.zindex=-1;然后，我在editor组件中，设置它style：w-e-text-container{z-index: 0 !important;};
这个操作提升了编辑器编辑区域的层级，至此，问题得以解决。
我提出这个问题，希望能有大佬给出明确的回答。
同时，为了给遇到和我同样问题的道友一个能够解决问题的方法。

**Comment by wangfupeng1988** [comment]
去提交 issue 吧，按照模板来。
